src/main/java/oc/wh40k/units/im/IMDominionSquad.java

```diff
@@ -44,7 +49,11 @@ public void refreshen() {
         if (squad.getModelle() <= 5)
             power = 5;
         else if (squad.getModelle() <= 10)
-            power = 9;
+            power = 7;
+        
+        bolters.setMaxAnzahl(squad.getModelle() - 1 - special.getAnzahl());
+        bolters.setAnzahl(0, bolters.getMaxAnzahl());
+        special.setMaxAnzahl(Math.min(4, squad.getModelle() - 1 - (simulacrum.isSelected() ? 1 : 0)));
```

Comment:
I have seen this block multiple times. create a function that does this?

Reply:
Differs depending on the squad times. It's either heavy, special or a combination.